### ISSUE TITLE
Errant, Non-Joypad Inputs when Pro Controller Is Connected

### ISSUE THREAD

**Issue by Sage-of-Mirrors** [ToadKing/switch-pro-x#1]
Hello,

Excuse me if this is a ViGEmBus issue, but I'm experiencing a problem where various things will go crazy when I run switch-pro-x. Like constant clicking even when I'm not clicking, Magnifer opening itself and magnifying to the maximum. I can even increase or decrease my system's volume with the left control stick. It stops when I unplug the Pro Controller. It seems like the system isn't reading the controller right and thinks it's doing all these bizarre inputs.

I'm on Windows 10, 1703, x64. I installed the x64 versions of HidGuardian and ViGEmBus 1.8, as you suggested. HidCerberus is running, but I might add that it's the version from the GitHub, which I believe is on 1.10. I'm not sure if that is an issue.

**Comment by ToadKing** [comment]
you need to use it in combination with the hidguardian driver as well.

**Comment by ToadKing** [comment]
Did you first manually block the Switch Pro Controller? The program does not do that automatically yet.

**Comment by Sage-of-Mirrors** [comment]
Do you mind if I ask how to do that? [I did this,](http://i.imgur.com/NwQp4Rl.png) but I'm not sure it's correct.

**Comment by ToadKing** [comment]
That should work. After a replug it should all work correctly. If not then maybe the driver had some changes.

**Comment by Sage-of-Mirrors** [comment]
I figured out what the issue is. For some reason, the Pro Controller itself doesn't get removed from the list of gamepads when it becomes one. So a valid virtual Xbox 360 controller is created and mapped to the Pro Controller, but there's still a super buggy Pro Controller gamepad that always registers as pressing buttons.

Windows shows that all of the buttons do indeed work, but for some reason when I tried to test inputs via Dolphin, it won't read certain buttons. I'll look into it, but for now, this is fixed. Thank you!
